app/policies/upcoming/manage/nil_class_policy.rb

```diff
@@ -0,0 +1,18 @@
+class Upcoming::Manage::NilClassPolicy < Upcoming::ApplicationPolicy
```

Comment:
I like using null objects like this - would this play in if someone creates a policy with a nil object?

Reply:
Yup.